src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/commands.py

```diff
@@ -221,6 +221,7 @@ def load_command_table(self, _):
         g.custom_command('identity assign', 'assign_identity')
         g.custom_show_command('identity show', 'show_identity')
         g.custom_command('identity remove', 'remove_identity')
+        g.custom_command('devops-build', 'create_devops_build')
```

Comment:
do we need other operation such as `show/list/delete/stop`? If yes, then `devops-build` should be a category with commands going under

Reply:
adding @ColbyTresness for his input.

Reply:
Good idea in principle, but we don't plan to add anything other than create functionality here due to limitations of the DevOps API, so I think it's best to leave it the way we have it. 

@dolko can provide an example of another command which does something similar already.

Reply:
I agree with @yugangw-msft, having a devops build as a category by itself will make it less confusing to the customers, even if it currently only have a create operation. When the api become richer it will also allow us to easily add the new capabilities in the future

Reply:
I agree with @yugangw-msft, having a devops build as a category by itself will make it less confusing to the customers, even if it currently only have a create operation. When the api become richer it will also allow us to easily add the new capabilities in the future